.github/workflows/build_and_publish_docs.yml

```diff
@@ -10,11 +10,16 @@ permissions:
 
 jobs:
   build-and-publish-docs:
-    runs-on: macos-12
+    runs-on: macos-14
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Setup Xcode 15.3
+        uses: maxim-lobanov/setup-xcode@60606e260d2fc5762a71e64e74b2174e8ea3c8bd # v1.6.0
+        with:
+          xcode-version: 15.3
+
       - name: Build docs
         run: |
           xcodebuild docbuild \
```

Comment:
Just to make sure, have you tried running it locally just to make sure everything works with the latest xcode?

Reply:
Yes, it worked
<img width="618" alt="image" src="https://github.com/software-mansion/starknet.swift/assets/54181625/85ae56f2-048c-44ef-a2e1-f221000dac33">